diff --git a/node_modules/@shopify/cli-hydrogen/dist/lib/mini-oxygen/workerd.js b/node_modules/@shopify/cli-hydrogen/dist/lib/mini-oxygen/workerd.js
index da14730..5969840 100644
--- a/node_modules/@shopify/cli-hydrogen/dist/lib/mini-oxygen/workerd.js
+++ b/node_modules/@shopify/cli-hydrogen/dist/lib/mini-oxygen/workerd.js
@@ -48,7 +48,7 @@ async function startWorkerdServer({
   const miniOxygen = createMiniOxygen({
     debug,
     port: appPort,
-    host: "localhost",
+    host: "0.0.0.0",
     liveReload: watch,
     requestHook: logRequestLine,
     inspectorPort: publicInspectorPort,
